docs/uml/macros.iuml

```diff
@@ -420,7 +418,7 @@ skinparam defaultMonospacedFontName Andale Mono
 !endif
 
 !ifdef SHOW_INSTANCE_CREATION && SHOW_CONSTRUCTOR
-  ContractInstanceCreation o-- "1" Constructor
+  ContractInstanceCreation --o "1" Constructor
```

Comment:
I think this arrow was correct behavior - the creation has the constructor, not the other way around